data/strings/types_strings.txt

```diff
@@ -471,6 +477,7 @@
     fa = خودپرداز
     fi = Pankkiautomaatti
     fr = GAB
+    hi = स्वचालित गणक मशीन
```

Comment:
almost no one uses or even understands the literal Hindi translation, but pretty much everyone uses the phonetic version of "ATM"
```suggestion
    hi = एटीएम
```